operator/internal/validation/openshift/alertingrule.go

```diff
@@ -33,6 +33,22 @@ func AlertingRuleValidator(_ context.Context, alertingRule *lokiv1.AlertingRule)
 					err.Error(),
 				))
 			}
+
+			if err := validateRuleLabels(rule.Labels); err != nil {
+				allErrs = append(allErrs, field.Invalid(
+					field.NewPath("Spec").Child("Groups").Index(i).Child("Rules").Index(j).Child("Labels"),
```

Comment:
The field path in the error should contain the names of the YAML/JSON fields and not the names of our Go attributes, also some chained calls can be combined:

```suggestion
					field.NewPath("spec", "groups").Index(i).Child("rules").Index(j).Child("labels"),
```

I realized this during another PR to `rulerconfig.go` but the other validators have not been updated yet.